### PR TITLE
Added C369087 - Orders| Results List | Verify that value in PO number…

### PR DESCRIPTION
**Pull request by KazachenkaTatsiana**
… and POL number columns are hyperlinks

https://foliotest.testrail.io/index.php?/cases/view/369087
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5921/console

image

### CODE REVIEW

**Review by IhorBohdan — APPROVED** [review]

File: cypress/support/fragments/orders/orderDetails.js [hunk 1]
```diff
@@ -232,4 +234,17 @@ export default {
       );
     }
   },
+
+  verifyOrderListIncludeLinkExists(linkName) {
```
Comment (manowarqqq) [inline cypress/support/fragments/orders/orderDetails.js:238]: use one verb either Include or Exists, not orderList but Order or OrderRow or smthg
lise verifyOrderItemIncludesLink()\
btw this method is not used

File: cypress/support/fragments/orders/orderLines.js [hunk 2]
```diff
@@ -1968,4 +1969,47 @@ export default {
       body: orderLine,
     });
   },
+  verifyPOlineListIncludeLinkExists: (POlinenumber) => {
```
Comment (manowarqqq) [inline cypress/support/fragments/orders/orderLines.js:1972]: use one verb either Include or Exists verifyPOlineListIncludesLink()

File: cypress/support/fragments/orders/orderLines.js [hunk 3]
```diff
@@ -1968,4 +1969,47 @@ export default {
...
+  },
+
+  verifyOrderTitlePOL(title) {
+    cy.expect(
+      orderLineDetailsPane.find(paneHeaderOrderLinesDetailes).has({ text: including(title) }),
+    );
+  },
+
+  selectStatusInSearchOrderLine: (orderStatus) => {
+    cy.wait(5000);
```
Comment (manowarqqq) [inline cypress/support/fragments/orders/orderLines.js:1987]: Why do you need  so long timeout?

File: cypress/support/fragments/orders/orderLines.js [hunk 4]
```diff
@@ -1968,4 +1969,47 @@ export default {
...
+    cy.do(Accordion({ id: 'receiptStatus' }).clickHeader());
+    switch (orderStatus) {
+      case 'Awaiting receipt':
+        cy.do(Checkbox({ id: 'clickable-filter-receiptStatus-awaiting-receipt' }).click());
```
Comment (manowarqqq) [inline cypress/support/fragments/orders/orderLines.js:1991]: better use Checkbox().checkIfNotSelected()

**Review by manowarqqq — APPROVED** [review]

**Pull request merged**